helm-chart/templates/_helpers.tpl

```diff
@@ -18,14 +18,14 @@
 {{- if not .Values.registry.disablePublicImages }}bitnami{{ else }}{{ .Values.registry.server }}{{ end }}
 {{- end }}
 
-{{- define "ubuntu.image-server" -}}
-{{- if not .Values.registry.disablePublicImages }}{{ else }}{{ .Values.registry.server }}/{{ end }}
-{{- end }}
-
 {{- define "curl.image-server" -}}
 {{- if not .Values.registry.disablePublicImages }}curlimages{{ else }}{{ .Values.registry.server }}{{ end }}
 {{- end }}
 
+{{- define "alpine.image-server" -}}
+{{- if not .Values.registry.disablePublicImages }}{{ else }}{{ .Values.registry.server }}{{ end }}
```

Comment:
You'll need a `/` if a user server is being used (see ubuntu or nats). This is because it'll be `their-server.com/alpine`, and they don't provide the `/`.